src/character.cpp

```diff
@@ -2306,3 +2282,63 @@ std::string Character::extended_description() const
 
     return replace_colors( ss.str() );
 }
+
+template <typename Getter>
+float Character::calc_mutation_value( Getter getter ) const
```

Comment:
Lambdas are temporary, cache would replace it.
Currently no evidence, but calculating this is linear with trait count.

You're right, it's not necessary. I'll add a string argument getter (something like `get_mutation_value( std::string what )` since it would stay useful even with the cache in place.